src/sql/workbench/contrib/dataExplorer/browser/extensionActions.ts

```diff
@@ -17,6 +17,7 @@ export const IMPORT_COMMAND_ID = 'dataExplorer.flatFileImport';
 export const SCHEMA_COMPARE_COMMAND_ID = 'dataExplorer.schemaCompare';
 export const GENERATE_SCRIPTS_COMMAND_ID = 'dataExplorer.generateScripts';
 export const PROPERTIES_COMMAND_ID = 'dataExplorer.properties';
+export const IMPORT_DATABASE_COMMAND_ID = 'dataExplorer.importDatabase';
```

Comment:
should this be `sqlDatabaseProjects.importDatabase`? (I don't know this area of the code, so just double-checking)

Reply:
It should be dataExplorer only, since it is under that context. sqlDatabaseProjects would probably for Import (and not ImportDatabase).